ONSdigital/eq-author-app
# Ear 1154 metadata dialog fix

> ### BEFORE MAKING YOUR PR
>
> Please ensure:
>
> - There are no linting errors, all tests must pass
> - PR is named after JIRA ticket number e.g. EAR-###
> - **Accesibility** checks are completed:
>   - Elements have discernible and consistent focus states
>   - Elements can be navigated to and used by just a keyboard
>   - Elements and text can be read out by a screen reader, and the descriptions are understandable
> - Your feature / bug fix works across **GCP** and **AWS** (where appropriate)
>   - Are modifications to eq-publisher-v3 required?
>   - Are modifications to the Firestore data source required?

---

### What is the context of this PR?

https://collaborate2.ons.gov.uk/jira/browse/EAR-1154

Fix bug preventing metadata search from working

Searching for text in the metadata table's key column caused all of the metadata options to be displayed after hovering over an option

### How to review

- Open metadata page
- Search for a valid metadata key in the key column (example - searching for "re" returns multiple results)
- Hover over a metadata option
- Check that hovering over the metadata option does not display metadata options that do not meet the search criteria (the list of filtered metadata does not get expanded)

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
